### PR TITLE
[EventGrid] Regenerate code using the latest rest commit sha

### PR DESCRIPTION
**Pull request by mssfang**
- Added new Api Management service system events, `ApiManagementApiCreatedEventData`, `ApiManagementApiDeletedEventData`, 
  `ApiManagementApiReleaseCreatedEventData`, `ApiManagementApiReleaseDeletedEventData`, 
  `ApiManagementApiReleaseUpdatedEventData`, `ApiManagementApiUpdatedEventData`, `ApiManagementProductCreatedEventData`,
  `ApiManagementProductDeletedEventData`, `ApiManagementProductUpdatedEventData`,
  `ApiManagementSubscriptionCreatedEventData`, `ApiManagementSubscriptionDeletedEventData`,
  `ApiManagementSubscriptionUpdatedEventData`,`ApiManagementUserCreatedEventData`, `ApiManagementUserDeletedEventData`,
  `ApiManagementUserUpdatedEventData`. 
- Added a new Media service system event, `MediaLiveEventChannelArchiveHeartbeatEventData`.
- Added a new Communication service system event, `AcsUserDisconnectedEventData`.

### CODE REVIEW

File: sdk/eventgrid/azure-messaging-eventgrid/src/main/java/com/azure/messaging/eventgrid/systemevents/MediaLiveEventIngestHeartbeatEventData.java [hunk 1]
```diff
@@ -8,8 +8,8 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 
 /**
- * Ingest fragment dropped event data. Schema of the data property of an EventGridEvent for a
- * Microsoft.Media.LiveEventIngestHeartbeat event.
+ * Ingest heartbeat event data. Schema of the data property of an EventGridEvent for a
+ * Microsoft.Media.LiveEventIngestHeartbeatEventData event.
  */
 @Immutable
 public final class MediaLiveEventIngestHeartbeatEventData {
```
Comment (mssfang) [inline sdk/eventgrid/azure-messaging-eventgrid/src/main/java/com/azure/messaging/eventgrid/systemevents/MediaLiveEventIngestHeartbeatEventData.java:15]: is this entire class need to be reverted?

File: sdk/eventgrid/azure-messaging-eventgrid/src/main/java/com/azure/messaging/eventgrid/implementation/SystemEventMappingNames.java [hunk 2]
```diff
@@ -467,17 +539,17 @@
...
             "Microsoft.Media.LiveEventIncomingVideoStreamsOutOfSync";
 
     /**
-     * Ingest fragment dropped event data. Schema of the data property of an EventGridEvent for a
-     * Microsoft.Media.LiveEventIngestHeartbeat event.
+     * Ingest heartbeat event data. Schema of the data property of an EventGridEvent for a
+     * Microsoft.Media.LiveEventIngestHeartbeatEventData event.
      */
-    public static final String MEDIA_LIVE_EVENT_INGEST_HEARTBEAT = "Microsoft.Media.LiveEventIngestHeartbeat";
+    public static final String MEDIA_LIVE_EVENT_INGEST_HEARTBEAT = "Microsoft.Media.LiveEventIngestHeartbeatEventData";
```
Comment (mssfang) [inline sdk/eventgrid/azure-messaging-eventgrid/src/main/java/com/azure/messaging/eventgrid/implementation/SystemEventMappingNames.java:552]: breaking changes. Do we need to keep EventData suffix consistency? 
Another example, which is the new class added.

MEDIA_LIVE_EVENT_CHANNEL_ARCHIVE_HEARTBEAT, MediaLiveEventChannelArchiveHeartbeatEventData.class

File: sdk/eventgrid/azure-messaging-eventgrid/src/main/java/com/azure/messaging/eventgrid/SystemEventNames.java [hunk 3]
```diff
@@ -300,6 +392,13 @@
      */
     public static final String MEDIA_JOB_STATE_CHANGE = "Microsoft.Media.JobStateChange";
 
+    /**
+     * Channel Archive heartbeat event data. Schema of the data property of an EventGridEvent for a
+     * Microsoft.Media.LiveEventChannelArchiveHeartbeatEventData event.
+     */
+    public static final String MEDIA_LIVE_EVENT_CHANNEL_ARCHIVE_HEARTBEAT =
+        "Microsoft.Media.LiveEventChannelArchiveHeartbeatEventData";
```
Comment (mssfang) [inline sdk/eventgrid/azure-messaging-eventgrid/src/main/java/com/azure/messaging/eventgrid/SystemEventNames.java:400]: Suffix LiveEventChannelArchiveHeartbeat**EventData**, the EventData can be removed?

Comment (JoshLove-msft) [reply sdk/eventgrid/azure-messaging-eventgrid/src/main/java/com/azure/messaging/eventgrid/SystemEventNames.java:400]: Yeah I think this was fixed in a later commit. Can you make sure you are using the latest SHA?

File: sdk/eventgrid/azure-messaging-eventgrid/src/main/java/com/azure/messaging/eventgrid/implementation/SystemEventMappingNames.java (hunk 2 above)
Comment (JoshLove-msft) [reply sdk/eventgrid/azure-messaging-eventgrid/src/main/java/com/azure/messaging/eventgrid/implementation/SystemEventMappingNames.java:552]: This was fixed in a later commit - see other comment.

File: sdk/eventgrid/azure-messaging-eventgrid/src/main/java/com/azure/messaging/eventgrid/SystemEventNames.java (hunk 3 above)
Comment (mssfang) [reply sdk/eventgrid/azure-messaging-eventgrid/src/main/java/com/azure/messaging/eventgrid/SystemEventNames.java:400]: https://github.com/Azure/azure-sdk-for-java/pull/24482/files#diff-d116ed60612e9aae51f2b7fe7b2de9482e764e38fcf7010003b4f36746b11ad7R500

Regenerated with lastest SHA, but it still contains a EventData suffix. It is a new class.

File: sdk/eventgrid/azure-messaging-eventgrid/src/main/java/com/azure/messaging/eventgrid/implementation/SystemEventMappingNames.java (hunk 2 above)
Comment (mssfang) [reply sdk/eventgrid/azure-messaging-eventgrid/src/main/java/com/azure/messaging/eventgrid/implementation/SystemEventMappingNames.java]: Thanks. With the latest SHA commit, it is good.

**Review by JoshLove-msft — APPROVED** [review]

**Pull request merged**